src/Presentation/Detection/Fetching/Types/TPDetection.cs

```diff
@@ -16,8 +16,9 @@ public TPDetection()
 
             Field(detection => detection.Id, true).Description("This is the ID");
             Field(detection => detection.Score, true).Description("This is the Score");
-            Field(detection => detection.Timestamp, true).Description("This is the Timestamp");
             Field(detection => detection.Class, true).Description("This is the Class");
+            Field(detection => detection.TimestampFilter, true, typeof(TTimestampFilter))
+                .Description("This is the timestamp");
```

Comment:
Should we try finding a more descriptive description for all of those fields?

Reply:
done